### ISSUE TITLE
version update packages with version num and build num

### ISSUE THREAD

**Issue by beckermr** [regro/cf-scripts#760]
We are now shipping a package that repackages other conda packages. The version update for it is more complicated, involving the build number. Plus we can probably use conda to get the latest version. 

@CJ-Wright for viz

related to https://github.com/conda-forge/staged-recipes/pull/10651

**Comment by beckermr** [comment]
We'll need to update sha's based on selectors. IDK if this works already or not.

**Comment by CJ-Wright** [comment]
I don't think selectored shas are currently updated, since they don't appear when we render (unless you pass the needed info into the render)

**Comment by CJ-Wright** [comment]
I think the problems are:
1. the code in [update_upstream_verions](https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/update_upstream_versions.py) doesn't handle things other than version numbers, so we need a way to also check increments of the other numbers.
2. the new key value pair needs to be migrated in the version migrator, requiring its own regex strings

**Comment by beckermr** [comment]
Would it help to special case anaconda urls?

**Comment by CJ-Wright** [comment]
You could do that. We could also request extra metadata about what keys/jinja variables to increment, and use that to configure the code

**Comment by beckermr** [comment]
Looks like we special case other urls. Adding an anaconda package one seems like it should be workable.

**Comment by isuruf** [comment]
> We could also request extra metadata about what keys/jinja variables to increment, and use that to configure the code

That wouldn't work. The build numbers are just random numbers like 166, 281 from Intel.

**Comment by beckermr** [comment]
Yup @isuruf. We can use conda itself to get the latest package for conda package urls.
